create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils.py

```diff
@@ -57,3 +61,17 @@ def init_config(
 ) -> None:
     app['config'] = \
         get_config(config or ['-c', DEFAULT_CONFIG_PATH.as_posix()])
+
+
+{%- if cookiecutter.use_postgres == 'y' %}
+if __name__ == '__main__':
```

Comment:
I think this is a bad idea. In my opinion, you should divide these functions into different files.
```
utils/
  - __init__.py
  - common.py
  - wait_script.py
```